SocialFilter/sections/specific.txt

```diff
@@ -3642,8 +3642,8 @@ indiaglitz.com##.socialnetwork_counts
 car.ru##.share__title-v2
 finecooking.com##.article__actions__social
 thebell.io##app-share-buttons
-onecall2ch.com,chaos2ch.com,himasoku.com,akb48matomemory.com,blog.livedoor.jp,gehasoku.com##.content > nav.section-box > ul.plugin-box
-blog.livedoor.jp,chaos2ch.com,michaelsan.livedoor.biz###article_bottom_v2 + nav.section-box
+oniyomech.livedoor.biz,onecall2ch.com,chaos2ch.com,himasoku.com,akb48matomemory.com,blog.livedoor.jp,gehasoku.com##.content > nav.section-box > ul.plugin-box
+oniyomech.livedoor.biz,blog.livedoor.jp,chaos2ch.com,michaelsan.livedoor.biz###article_bottom_v2 + nav.section-box
```

Comment:
Probably better to apply to all `livedoor.biz` (merge with `michaelsan.livedoor.biz`)

Reply:
ok